avocado/core/status/repo.py

```diff
@@ -72,12 +76,13 @@ def _update_status(self, message):
             return
         if task_id not in self._status:
             self._status[task_id] = (status, time)
-            self._status_journal_summary.append((task_id, status, time))
+            self._status_journal_summary.append((task_id, status, time, 0))
```

Comment:
Yes, the docstring have to be updated. 

I can't use `_all_data` directly because I want to process the messages in an order as they were recorded into the `StatusRepo` from all running tasks but the `_all_data` sorts those messages by tasks. And when I pop some message from the `_status_journal_summary` I have to connect this message with data in `_all_data` and that is not an easy task without the index. 

I was planing to leave all the message data in `_status_journal_summary` but I am not quite sure what consequences will have such change, so I decided to use an index.